prov/sockets/src/sock_av.c

```diff
@@ -527,7 +527,7 @@ int sock_av_open(struct fid_domain *domain, struct fi_av_attr *attr,
 
 		_av->table_hdr = mmap(NULL, table_sz, PROT_READ | PROT_WRITE,
 					MAP_SHARED, _av->shared_fd, 0);
-		_av->attr.map_addr = _av->table_hdr + sizeof(_av->table_hdr);
+		_av->attr.map_addr = _av->table_hdr + sizeof(struct sock_av_table_hdr);
```

Comment:
Apparently, shared_av test fails when I use <code>_av->attr.map_addr = _av->table_hdr + 1</code> instead of <code>_av->attr.map_addr = _av->table_hdr + sizeof(_av->table_hdr) </code>. Working on it.